portal/src/main/webapp/WEB-INF/jsp/visualize.jsp

```diff
@@ -426,6 +434,17 @@ window.loadReactApp({ defaultRoute: 'results' });
                 position: {my:'left top',at:'right bottom', viewport: $(window)}
             }
         );
+        $("#fusion-data-result-tab").qtip(
+            {
+                content: {text: 
+                    "Fusion genes, hybrid gene formed from two previously separate genes. It can occur as a result of: " +
+                    "translocation, interstitial deletion, or chromosomal inversion *CHANGE THIS PLEASE*"},
```

Comment:
What about: "Fusion details, including the exact exons where the fusion between two genes has occured, frame effects and annotations", or something similar